iquip/apps/explorer.py

```diff
@@ -130,7 +137,7 @@ def _addFile(self, experimentList: List[str], parent: Union[QTreeWidget, QTreeWi
 
         Args:
             experimentList: The list of files under the parent path.
-            parent: See FileFinderThread class.
+            parent: See _FileFinderThread class.
```

Comment:
Since we changed `parent` -> `widget`, it should be applied here as well.

Reply:
Oh, I thought remaining it as `parent` doesn't make a confusion in this method, so I left it as it is.
But if you want to apply it, I will do that.